src/component/AddItem.js

```diff
@@ -32,13 +52,29 @@ const AddItem = ({ firestore }) => {
   };
 
   const handleOptionChange = event => {
-    setFrequency(event.target.value);
+    setNumberOfDays(event.target.value);
+  };
+
+  // function triggered at handleSubmit -
+  // normalizes item name so that it has all lowercase
+  // letters and no special characters (spaces ok)
+  const normalizeName = name => {
+    name = name.toLowerCase();
+    let normalizedName = "";
+    let alpha = "abcdefghijklmnopqrstuvwxyz ";
+    for (let i = 0; i < name.length; i++) {
+      if (alpha.includes(name[i])) {
+        normalizedName += name.slice(i, i + 1);
+      }
+    }
+    return normalizedName;
   };
 
   //   Trigger addItem function when "Add Item" button is clicked
   const handleSubmit = event => {
     event.preventDefault();
-    addItem(name, frequency);
+    let normalizedName = normalizeName(name);
+    addItem(normalizedName, numberOfDays);
     setName("");
```

Comment:
Would it be possible to only clear the name from the field if it passes validation?